### ISSUE TITLE
Ensure `prefixJwtVcPath` can be used when generating authz requests during oid4vci/oid4vp

### ISSUE THREAD

**Issue by dlongley** [digitalbazaar/bedrock-vc-delivery#114]
See:

https://github.com/digitalbazaar/bedrock-vc-delivery/blob/f3d2e4f305df6bfefe0cb659343c21833f5ccf80/lib/oid4/oid4vp.js#L87
https://github.com/digitalbazaar/oid4-client/blob/v4.0.0/lib/oid4vp.js#L301

For now, accepted cryptosuites / accepted envelopes as specified in the VPR could be used to decide whether to use `prefixJwtVcPath` (and if unspecified, we can continue with the present default). Ideally, multiple options would be supported as in this example:

https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#section-5.1-6

We might also need a specific behavioral hack based on required credential issuer / credential type to support interop in the near term with special features like using the `pattern` filter (but used as a `const` is used) might be required.

For an example VPR that includes both accepted cryptosuites and envelopes:

```js
{
  "type": "QueryByExample",
  "credentialQuery": {
    "reason": "Please present your PermanentResidentCard or Verifiable Credential(s) to complete the verification process.",
    "example": {
      "@context": [
        "https://www.w3.org/2018/credentials/v1",
        "https://w3id.org/citizenship/v1"
      ],
      "type": ["PermanentResidentCard"]
    },
    "acceptedCryptosuites": [
      "Ed25519Signature2020",
      "eddsa-rdfc-2022",
      "ecdsa-rdfc-2019",
      "bbs-2023",
      "ecdsa-sd-2023"
    ],
    "acceptedEnvelopes": [
      "application/jwt"
    ]
  }
}
```

**Comment by dlongley** [comment]
Implementation of this feature: https://github.com/digitalbazaar/oid4-client/issues/34

Followed by an upgrade to use it might solve this indirectly.

**Comment by dlongley** [comment]
Addressed by #115.
